lib/cms/repo.ex

```diff
@@ -182,6 +217,29 @@ defmodule CMS.Repo do
     end
   end
 
+  # BEGIN PAGE CACHING #
+
+  @behaviour Nebulex.Caching.KeyGenerator
+
+  @impl true
+  def generate(_, _, [path, %Plug.Conn.Unfetched{aspect: :query_params}]) do
+    "/cms/#{String.trim(path, "/")}"
+  end
+
+  def generate(_, _, [path, params]) do
+    "/cms/#{String.trim(path, "/")}" <> params_to_string(params)
```

Comment:
This struck me as a good idea, but now that I see it (I ran the smoke tests against this branch locally to cache a lot of pages) I'm kinda meh on it? I'm leaning toward not keying by query params... at least for CMS content?

<img width="559" alt="image" src="https://github.com/mbta/dotcom/assets/2136286/a6bc47cf-be79-4f00-a34b-58745c36eead">

Reply:
The problem is that we don't know if query params will result in different content. I think it would be better to miss the cache than serve incorrect content.